packages/chakra-components/src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,187 @@
+import { ButtonGroup, ButtonGroupProps, ButtonProps, InputProps, Text } from '@chakra-ui/react'
+import { ReactElement, useMemo } from 'react'
+import { Link as RouterLink } from 'react-router-dom'
+import { useLocalize, usePagination, useRoutedPagination } from '@vocdoni/react-providers'
+import { PaginationResponse } from '@vocdoni/sdk'
+import { useMultiStyleConfig, chakra } from '@chakra-ui/system'
+import { EllipsisButton } from './EllipsisButton'
+import { PaginatorButton } from './PaginatorButton'
+
+export type PaginationProps = ButtonGroupProps & {
+  maxButtons?: number | false
+  buttonProps?: ButtonProps
+  inputProps?: InputProps
+} & PaginationResponse
+
+type PaginatorButtonProps = {
+  page: number
+  currentPage: number
+} & ButtonProps
+
+const PageButton = ({ page, currentPage, ...rest }: PaginatorButtonProps) => (
+  <PaginatorButton isActive={currentPage === page} {...rest}>
+    {page + 1}
+  </PaginatorButton>
+)
+
+const RoutedPageButton = ({ page, currentPage, to, ...rest }: PaginatorButtonProps & { to: string }) => (
+  <PaginatorButton as={RouterLink} to={to} isActive={currentPage === page} {...rest}>
+    {page + 1}
+  </PaginatorButton>
+)
```

Comment:
We gotta add styling capabilities to these components (use the same for both, call it `PageButton` for example). Review all visible components have their style key defined, and allow receiving props (this one allows that already, but we should add theme to these components anyway).